templates/zerver/integrations.html

```diff
@@ -1639,7 +1639,8 @@
           <p>To configure Zulip notifications for a particular Redmine project,
             visit the project's Settings page.</p>
           <p>
-            In either case, fill out the bot email address and API key, and the
+            In either case, fill out zulip server by <code>{{ external_api_path_subdomain }}</code>,
```

Comment:
I'm not sure If I understood you correctly. For now generic class `Client` supports both ways with and without `/api`. I also see that only one place where we're using `ZULIP_SITE` is  passing it to `Client` class.
So it should work in both cases - Did I miss something?